e2e/rdr-e2e.sh

```diff
@@ -1,4 +0,0 @@
-#!/bin/bash
```

Comment:
As this is invoked from the Makefile (see `e2e-rdr` Makefile target) from a merge POV this needs to exist, even if a do nothing file.

I was attempting to make changes here and run the skeleton, see: 
- https://github.com/RamenDR/ramen/actions/runs/8801029790/job/24153462114
- https://github.com/RamenDR/ramen/pull/1355/commits/97c017fc509063432961109f0e939ba803066d4e

Strangely I did not see logs in the GH action run, maybe I am missing something, a local run was succesful.